src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -20,48 +25,57 @@ const concatDescription = (position: SyPersonioJob): string => {
   );
 };
 
+/**
+ * Create and write structured data based on https://schema.org/JobPosting
+ *
+ * See the Google guide for additional insights:
+ * https://developers.google.com/search/docs/advanced/structured-data/job-posting#structured-data-type-definitions
+ */
 export const CareerDetailsStructuredData = ({
   position,
 }: CareerDetailsStructuredDataProps) => {
-  const employmentType = convertScheduleToEmploymentType(position.schedule);
-
-  const data = {
-    '@context': 'https://schema.org/',
+  const structuredData: WithContext<JobPosting> = {
+    '@context': 'https://schema.org',
     '@type': 'JobPosting',
     title: position.name,
     datePosted: position.createdAt,
     description: concatDescription(position),
     identifier: {
       '@type': 'PropertyValue',
       name: 'Satellytes Digital Consulting GmbH',
-      value: '338504-41230',
+      value: position.jobId,
     },
     hiringOrganization: {
       '@type': 'Organization',
       name: 'Satellytes Digital Consulting GmbH',
       sameAs: 'http://www.satellytes.com',
-      logo: 'https://we-are-hiring.cdn.personio.de/logos/41230/social/6ca8f29950517ac39fcae0d6545a1fe6.png',
     },
-
     jobLocation: {
       '@type': 'Place',
       address: {
         '@type': 'PostalAddress',
-        streetAddress: 'Sendlinger Stra\u00dfe 52',
-        addressLocality: 'M\u00fcnchen',
+        streetAddress: 'Sendlinger Straße 52',
+        addressLocality: 'München',
         addressRegion: 'Bayern',
         postalCode: '80331',
         addressCountry: 'DE',
       },
     },
     jobLocationType: 'TELECOMMUTE',
-    employmentType: employmentType,
+    applicantLocationRequirements: {
+      '@type': 'AdministrativeArea',
+      sameAs: 'http://www.wikidata.org/entity/Q458',
+      name: 'European Union',
+    },
+    employmentType: normalizeJobScheduleFormat(position.schedule),
     directApply: true,
   };
 
   return (
     <Helmet>
-      <script type="application/ld+json">{JSON.stringify(data)}</script>
+      <script type="application/ld+json">
+        {JSON.stringify(structuredData)}
+      </script>
```

Comment:
prettier did that I think. I wonder what's missing for @milljoniaer that this wasn't triggered?

Reply:
What is your Point here? I am not sure what to do here? 😅

Reply:
A nitpick. The format changed for some reason, but when I look at the file now it's  just a fresh addition so I don't know why I wrote that back then 😬